Author: Bernhard M. Wiedemann <bwiedemann suse.de>
Subject: Declare build dependency
Date: 2022-09-24

Declare build dependency to ensure consistent make execution order.
See https://reproducible-builds.org/ for why this is good.

Without this patch, parallelism would make a difference in
the produced config.o

Index: mirrormagic-3.0.0/src/Makefile
===================================================================
--- mirrormagic-3.0.0.orig/src/Makefile
+++ mirrormagic-3.0.0/src/Makefile
@@ -271,6 +271,8 @@ $(TIMESTAMP_FILE): $(SRCS) $(RNDLIBS)
 	@echo '#define SOURCE_DATE_STRING "$(SOURCE_DATE_STRING)"' \
 	> $(TIMESTAMP_FILE)
 
+config.o: $(TIMESTAMP_FILE)
+
 $(ICON):
 #	$(CONVERT) $(ICON32X32) $(CONVERT_ICON_ARGS) $(ICONBASE).ico
 	$(CONVERT) $(ICON_PATH)/*.png $(CONVERT_ICON_ARGS) $(ICONBASE).ico
